tensorflow/core/kernels/dynamic_stitch_op.cc

```diff
@@ -95,46 +205,13 @@ class DynamicStitchOp : public OpKernel {
     }
     Tensor* merged = nullptr;
     OP_REQUIRES_OK(c, c->allocate_output(0, result_shape, &merged));
-
     // TODO(jeff): Currently we leave uninitialized any portions of
     // merged that aren't covered by an index in indices.  What should we do?
     if (first_dim_size > 0) {
```

Comment:
Can't this be `result_shape.num_elements() > 0` to exit early if other dimensions are zero too?